### ISSUE TITLE
Previewing PDF files as images doesn't work any more due pdftoppm/pdftopng version 4.0 droped options.

### ISSUE THREAD

**Issue by moisestohias** [ranger/ranger#2782]
Previewing PDF files as images doesn't work any more due pdftoppm/pdftopng version 4.0 droped options.
pdftoppm and pdftopng version 4.0 and above don't have most of the options used in the scoop.sh file. Namely:
`-scale-to-x`
`-scale-to-y`
`-singlefile`
`-jpeg`
`-tiffcompression jpeg`
pdftoppm can be replaced by pdftopng.
Also both pdftoppm & pdftopng add the page number suffix 0001.ppm|0001.png making the IMAGE_CACHE_PATH wrong.

```bash
## PDF (comment these lines for PDF image preview instead of txt)
application/pdf)
    # Options other than -f/l only works for version 3.0 not 4.0
    pdftoppm -f 1 -l 1 \ 
             -scale-to-x "${DEFAULT_SIZE%x*}" \
             -scale-to-y -1 \
             -singlefile \
             -jpeg -tiffcompression jpeg \
             -- "${FILE_PATH}" "${IMAGE_CACHE_PATH%.*}" \
        && exit 6 || exit 1
```

**Comment by saipavanc** [comment]
Yes, I have just noticed that no new pdf previews are being generated in the last couple of days. Is there a resolution yet? Is it possible to replace `pdftoppm` with `pdftopng` with a properly edited `scope.sh` file?

**Comment by moisestohias** [comment]
This can be fixed by droping all the used options, and accounting for the suffix added by pdftoppm/pdftopng (-0001).

**Comment by toonn** [comment]
Where is this version coming from? [The Poppler program seems to still have these flags](https://github.com/freedesktop/poppler/blob/master/utils/pdftoppm.cc#L159-L160).

**Comment by nekomi2** [comment]
I'm also getting this issue where pdftoppm isn't generating previews, is there a solution?
